### ISSUE TITLE
Big number text-only variant is announced as two separate items by VoiceOver and TalkBack

### ISSUE THREAD

**Issue by hannalaakso** [alphagov/govuk_publishing_components#5563]
## What
iOS and Mac VoiceOver, and Android TalkBack currently read the value (e.g., "82") and the label (e.g., "Open consultations") of the [big number component text-only version](https://components.publishing.service.gov.uk/component-guide/big_number) as two separate elements. They should be wrapped/marked up so the entire phrase is announced as one single item or link.

[We fixed the issue for JAWS and NVDA recently](https://github.com/alphagov/govuk_publishing_components/pull/5458), there's more detail on that PR on what we looked at. VoiceOver and TalkBack also read the link version of the big number as one item. However, we didn't find a solution for VoiceOver and TalkBack when big number isn't a link. 

It might be that it's not possible to fix this for all the combinations of screen readers we support. In that case we might want to look at redesigning the component to support it being announced as one item.

## Why
This issue was raised in the 2025 GOV.UK accessibility audit:

> When a text within a semantic element such as a link or a heading is split across multiple child elements, this can mean that instead of being relayed all in one go, it is relayed in pieces, with the ‘link’ or ‘heading’ role relayed after each piece, as if they were separate links or separate headings. For example, screen reader users may encounter only ‘66’, followed by ‘Open consultations’ and assume that these are separate links with different destinations.
